src/test/java/bio/terra/app/controller/AdminApiControllerTest.java

```diff
@@ -155,16 +157,17 @@ void testAdminRetrieveSnapshotNotFound() throws Exception {
     mvc.perform(get(ADMIN_SNAPSHOTS_ENDPOINT, MODEL_ID)).andExpect(status().isNotFound());
   }
 
-  @Test
-  void testAdminInheritStewardEnable() throws Exception {
+  @ParameterizedTest
+  @ValueSource(booleans = {true, false})
+  void adminInheritSteward(boolean inheritSteward) throws Exception {
     String jobId = "jobId";
     when(datasetService.retrieveDatasetSummary(MODEL_ID))
-        .thenReturn(new DatasetSummaryModel().id(MODEL_ID).inheritSteward(false));
-    when(datasetService.enableInheritSteward(eq(MODEL_ID), any())).thenReturn(jobId);
+        .thenReturn(new DatasetSummaryModel().id(MODEL_ID).inheritSteward(!inheritSteward));
+    when(datasetService.setInheritSteward(eq(MODEL_ID), eq(inheritSteward), any()))
```

Comment:
I think the last arg is known here
```suggestion
    when(datasetService.setInheritSteward(MODEL_ID, inheritSteward, TEST_USER))
```